src/main/java/com/sericulture/authentication/repository/UserInfoRepository.java

```diff
@@ -10,5 +10,5 @@
 public interface UserInfoRepository extends JpaRepository<UserInfo, Integer> {
     Optional<UserInfo> findByUsername(String username);
     Optional<UserInfo> findByEmailID(String emailId);
-    Optional<UserInfo> findByEmailIDAndRoleId(String emailId,long roleId);
+   //Optional<UserInfo> findByEmailIDAndRoleId(String emailId,long roleId);
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 3, expected is 4, indentation should be the same level as line 12.